gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveCopyEntityHelper.java

```diff
@@ -293,8 +296,12 @@ protected static Path resolvePath(String pattern, String database, String table)
   private Table getTargetTable(Table originTable, Path targetLocation) throws IOException {
     try {
       Table targetTable = originTable.copy();
+      
       targetTable.setDbName(this.targetDatabase);
       targetTable.setDataLocation(targetLocation);
+      
+      HiveAvroCopyEntityHelper.updateAvroTableAttributes(targetTable);
```

Comment:
Can you wrap this into logic that checks whether it is an Avro table? Or add that logic in the method itself? We don't want to accidentally modify a non-Avro table.